Assignment/solution_student/index.html

```diff
@@ -5,27 +5,94 @@
-      </main>
+<!--enjoy this delightful lorem ipsum-->
 
-      <footer>
-        <p>&copy; FEWD6 & Friends, INC.</p>
+      <section class="content-wrapper clearfix">
+        <section class="articles">
+          <article>
+            <h2>How I implemented Relaxr in 2 weeks and changed my life.</h2>
+              <img class="article_images" src="images/blog_photo1.jpg">
+              <p>
+                <span class="first_character">L</span>
+                orizzle ipsizzle dolor pot amet, fo shizzle adipiscing elit. Nullizzle crackalackin velizzle, away volutpizzle, suscipit fo shizzle, rizzle vel, arcu. Pellentesque i saw beyonces tizzles and my pizzle went crizzle tortor. Etizzle we gonna chung away ut i'm in the shizzle. Get down get down quizzle arcu. Break it down sizzle, ipsum owned for sure , nulla purus crazy break yo neck, yall, izzle luctus mofo nulla izzle . Vivamizzle nizzle, tortor izzle varizzle shizznit, nibh doggy ultricizzle get down get down, izzle luctizzle bling bling shiz izzle dolizzle. Maurizzle aliquet, orci vel yippiyo dawg, sizzle augue luctus nisi, at rizzle enizzle izzle nis.</p>
+             <a href="#">Read More></a>
+          </article>
+
+          <hr>
+
+          <article>
+            <h2>I traveled across the globe thanks to Relaxr’s Excel automation. </h2>
+            <img class="article_images" src="images/blog_photo2.jpg">
+            <p>
+              <span class="first_character">L</span>
+              ivamizzle nec maurizzle the bizzle shut the shizzle up consectetizzle shiznit. Fo sizzle cool dope. Nam eu doggy egizzle lacizzle auctizzle feugiat. Praesent funky fresh viverra fo shizzle. Bizzle izzle arcu. pizzle enizzle, crackalackin shiznit, congue eu, dignissizzle stuff, libero. Nullizzle gangsta shizzle my nizzle crocodizzle non erizzle break it down shizzle my nizzle crocodizzle. Quisque black tortizzle, congue pulvinar, auctor a, bizzle sizzle amet, erat. Donec izzle check it out. Aliquizzle check out this brizzle, fo consectetizzle, own yo' izzle, consequat yippiyo, fo shizzle. Quisque a ipsizzle bling bling mi dawg vehicula. That's the shizzle accumsizzle sagittis ipsizzle. Pellentesque habitant owned tristique da bomb shizzle my nizzle crocodizzle netizzle et hizzle dope ac turpis stuff. 
+            </p>
+            <a href="#">Read More></a>
+          </article>
+        </section>
+
+        <section class="sidebar">
+          <h3>Categories</h3>
+            <ul>
+              <li>Success Stories</li>
+              <li>Stats</li>
+              <li>How Tos</li>
+              <li>Best Of</li>
+              <li>Business Tips</li>
+            </ul>
+
+          <h3>About Relaxr</h3>   
+            <p>Pizzle dizzle. Aliquizzle sagittizzle massa a check out this. Dope ante ipsizzle primis funky fresh faucibizzle stuff luctus izzle</p>
+            <a href="#">Learn More</a>
+        </section>
+
+        <img class="160x600" src="images/ad-unit.png">
```

Comment:
The reason the styles you tried to apply to this image didn't work is because you started the name of your class with a number, which is not allowed. Your class names should always start with a lower case letter, a "_" or a "-", though that's for more complex systems. You could also have just moved this `img` inside the `.sidebar` section and it would have been positioned properly too :)